README.md

```diff
@@ -359,6 +361,67 @@ When using flakes, switch to new configurations as you do for the
 whole system (e. g. `nixos-rebuild switch --flake <path>`) instead of
 using the `home-manager` command line tool.
 
+### For unprivileged setup
+
+If you would like to set up Home Manager as an unprivileged user:
+
+1.  Set up a flake with a `flake.nix` as follows:
+
+    ```Nix
+    {
+      description = "Home Manager configuration of Jane Doe";
+    
+      inputs = {
+        home-manager.url = "github:nix-community/home-manager";
+        # Optional, get in sync with the desired nixpkgs source.
+        # Use the right nixpkgs version according to the Home Manager release used.
+        nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+        home-manager.inputs.nixpkgs.follows = "nixpkgs";
+      };
+    
+      outputs = { home-manager, ... }:
+        let
+          system = "x86_64-linux";
+          username = "jdoe";
+        in {
+          homeConfigurations.${username} = home-manager.lib.homeManagerConfiguration {
+            configuration = import ./home.nix;
+            inherit system username;
+            homeDirectory = "/home/${username}";
+            # Update the state version as needed.
+            # See the changelog here:
+            # https://nix-community.github.io/home-manager/release-notes.html#sec-release-21.05
+            stateVersion = "21.05";
+            # # Pass extra arguments into home.nix using
+            # extraSpecialArgs = {
+            #   key = value;
+            # };
+          };
+      };
+    }
+    ```
+    *   If you would like to use the `release-21.05` branch, change the `home-manager` input url to `github:nix-community/home-manager/release-21.05` and `nixpkgs` url to `github:NixOS/nixpkgs/nixos-21.05`. Make sure to also update to the `stateVersion` option accordingly.
+    *   This example implies that the configuration is written in the file `home.nix` in the same directory level as `flake.nix`. You may change the path to `configuraion = path/to/your-home-config.nix` or specify the configuration directly.
```

Comment:
`configuraTion`

Reply:
this is basic nix stuff and even if you dont know nix, I think it's straightforward and the error message will tell you where it's looking for files. So I would remove it altogether.

Reply:
> The interesting bit is that home-manager accepts a --flake argument like this:
> 
> ```
> home-manager switch --flake ~/.config/nixpkgs#jdoe
> ```

A line about the `home-manager` cli was added at the end of the Flake section of the README recently.

> There is too much text explaining how flakes work IMO. There is an official documentation for that.

I wonder whether Home Manager is the first experience people have with Nix?
I think providing a simple explanation of the flake basics is good.

Documentation about the regular workflow (updating flake.lock inputs, etc.) is also good and should be kept.

> the rest could go to the wiki and be referenced from the README. Let's try to keep in check the README length.

I agree.